plan/physical_plan_builder.go

```diff
@@ -871,24 +871,16 @@ func (p *Union) convert2PhysicalPlan(prop *requiredProperty) (*physicalPlanInfo,
 	}
 	limit := prop.limit
 	childInfos := make([]*physicalPlanInfo, 0, len(p.children))
-	var count uint64
 	for _, child := range p.GetChildren() {
-		newProp := convertLimitOffsetToCount(prop)
-		newProp.props = make([]*columnProp, 0, len(prop.props))
-		for _, c := range prop.props {
-			idx := p.GetSchema().GetIndex(c.col)
-			newProp.props = append(newProp.props, &columnProp{col: child.GetSchema()[idx], desc: c.desc})
-		}
+		newProp := limitProperty(limit)
```

Comment:
Does this mean "limit" can be pushed to children without "order by" is pushed?
What about this case:
```
create t1 (a int);
insert t1 values (2), (1);
create t2 (a int);
insert t2 values (3), (4);
select * from t1 union all (select * from t2) order by a limit 1;
```
The correct result is `1`, but if limit is pushed to children, the result would be `2`.